stack.default.yml

```diff
@@ -17,16 +17,16 @@ configure:
   # setup:
   #   build:
   #     args:
-  #       exercism_config_version: 0.56.0
+  #       exercism_config_version: 0.60.0
```

Comment:
I might be wrong, but I believe that the setup gem is build using the version in the `Gemfile.lock` of this repo if this is left commented out. So we should also run `bundle update` as part of this PR to keep that comment in sync with the `Gemfile.lock`.

Reply:
Ah, I'll update! I'm always slightly confused about the setup bit as that was developed in my holiday :)

Reply:
I've updated.